package.json

```diff
@@ -79,24 +79,28 @@
   "dependencies": {
     "3box": "1.17.1",
     "@burner-wallet/burner-connect-provider": "^0.1.1",
-    "@daostack/client": "0.2.67",
-    "@dorgtech/daocreator-ui": "^1.0.8",
+    "@dorgtech/arc.js": "2.0.0-experimental.32",
+    "@dorgtech/daocreator-ui-experimental": "1.1.2",
     "@fortawesome/fontawesome-svg-core": "^1.2.10",
     "@fortawesome/free-brands-svg-icons": "^5.6.1",
     "@fortawesome/react-fontawesome": "^0.1.3",
     "@portis/web3": "^2.0.0-beta.43",
     "@sentry/browser": "^5.0.8",
     "@toruslabs/torus-embed": "^1.3.0",
     "@walletconnect/web3-provider": "^1.0.0-beta.37",
+    "apollo-link-retry": "^2.2.16",
     "axios": "^0.19.0",
     "bn.js": "^4.11.8",
     "classnames": "^2.2.5",
     "clone-deep": "^4.0.1",
     "css-loader": "^1.0.1",
     "disqus-react": "^1.0.5",
+    "dotenv": "^8.0.0",
+    "graphql-tag": "^2.10.3",
```

Comment:
why this is needed ?

Reply:
webpack files import dotenv. 7 files import graphql-tag. Previously the build was working because these were inside the package-lock. Failures occurred when the package-lock changed, and alchemy could no longer use these packages from the direct dependencies they were coming from.